components/03-modules/footer/_footer.scss

```diff
@@ -0,0 +1,172 @@
+$footer__bg-color                    : $gray-darker !default;
+$footer__bg-color--screen-l          : $gray-dark !default;
+$footer__font-color                  : $white !default;
+$footer__transition                  : $transition-base !default;
+
+$footer__newsletter-max-width        : 400px !default;
+$footer__newsletter-width--screen-l  : 30% !default;
+
+$footer__links-width--screen-l       : 70% !default;
+$footer__link-color-after-hover      : $color-primary !default;
+$footer__link-height-after-hover     : 8px !default;
+$footer__link-font-weight-after-hover: $font-weight-bold !default;
+
+$footer__bottom-bar-background       : $gray-darker !default;
+$footer__bottom-bar-color            : $gray !default;
+$footer__bottom-bar-font-size        : $font-size-small !default;
+
+
+.footer {
+    background-color: $footer__bg-color;
+    padding: $spacer--medium 0;
```

Comment:
I think there should also be a new internal variable for this padding.

Reply:
right, moved to variable